src/components/redux/greetingSlice.js

```diff
@@ -0,0 +1,38 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+// Define your async thunk
+export const fetchRandomGreeting = createAsyncThunk(
+  'greeting/fetchRandomGreeting',
+  async () => {
+    const response = await axios.get('http://localhost:3001/api/v1/random_greeting');
```

Comment:
- [ ] Great job so far @tomdan-ai However, there is currently a CORS error that is preventing your Frontend from working seamlessly with your Backend. So kindly fix this :wink: 
- Error Message
![Screenshot from 2023-08-26 22-48-22](https://github.com/tomdan-ai/hello-react-front-end/assets/102699958/3f3d0517-148b-4ae0-92aa-224db8233419)

Reply:
Thank you for the time taken to review, I appreciate👍
But the error your getting is unique to your device, because the work perfectly on my End 👇👇👇
**Only Linter errors ignored on GITHUB**
![Screenshot 2023-08-28 165909](https://github.com/tomdan-ai/hello-react-front-end/assets/88054834/9136a444-62ea-4985-b07b-55e768d78bbf)

**No Cors Error**
![Screenshot 2023-08-28 165944](https://github.com/tomdan-ai/hello-react-front-end/assets/88054834/75c370ef-1a34-4825-adfc-19d392164a79)